tests/platform_tests/test_link_down.py

```diff
@@ -10,12 +10,13 @@
 import logging
 import time
```

Comment:
can you please remove this? thanks
tests/platform_tests/test_link_down.py:11:1: F401 'time' imported but unused

Reply:
With adding 'time.sleep(MAX_TIME_TO_REBOOT)' at the beginning of test, we would need this import